register-service/src/main/resources/sql/20211118_08_create_slot_to_user_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE slot_to_user
+(
+    id      SERIAL,
+    slot_id BIGINT,
+    user_id BIGINT
+);
```

Comment:
Please, add constraints on slot and user tables

Reply:
will be added later in scope of https://github.com/javaGuruBY/jis8-project/issues/44